### DISCUSSION TITLE
User-Agents

### DISCUSSION THREAD

**Discussion by kski44** [tarampampam/random-user-agent#565] [General]
Hello,

Do you have a bulk list of current user agents from Desktop/Mobile/Chrome/Firefox?

Best regards

**Comment by tarampampam** [comment]
Hi there! Nope, this extension generates a new one each time, so it doesn't have something like a "list," but it does have patterns

**Comment by kski44** [comment]
Can you maybe do something like that as a new project? I need a list of 1000 current user agents

> Am 12.04.2024 um 16:12 schrieb Pаramtamtām ***@***.***>:
> 
> Hi there! Nope, this extension generates a new one each time, so it doesn't have something like a "list," but it does have patterns

**Reply by tarampampam** [reply]
You can take code from this project and genre as many user agents as you need - the code is open and that's very easy 😉

**Reply by kski44** [reply]
where can I find that?

**Reply by kski44** [reply]
How can I turn the code into a file that creates a bulk list for me?

**Reply by tarampampam** [reply]
https://github.com/tarampampam/random-user-agent/blob/3cc1fc1687a250a67eb76c514d8ea00b6ce9a401/src/useragent/generator.test.ts#L16
